docs/API.md

```diff
@@ -364,6 +364,8 @@ Options can be configured as a parameter to the [init()](./interfaces/tracer.htm
 | flushInterval   | -                                  | `2000`         | Interval in milliseconds at which the tracer will submit traces to the agent. |
 | flushMinSpans   | `DD_TRACE_PARTIAL_FLUSH_MIN_SPANS` | `1000`         | Number of spans before partially exporting a trace. This prevents keeping all the spans in memory for very large traces. |
 | -               | `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP` | -      | A regex to redact sensitive data from incoming requests' querystring reported in the `http.url` tag (matches will be replaced with `<redacted>`). Can be an empty string to disable redaction or `.*` to redact all querystring. **WARNING: this regex will execute for every incoming request on an unsafe input (url), please make sure you use a safe regex.** |
+| -               | `DD_TRACE_CLIENT_IP_HEADER_DISABLED` | `false`      | Whether to enable HTTP client IP reporting. Setting this to `true` will disable collection of the `http.client_ip` tag. |
```

Comment:
What's the use case for disabling this?

Reply:
Also, it would be more consistent with other features to use `_ENABLED` instead of `_DISABLED`.

Reply:
Yeah I definitely facepalmed when I saw the "Disabled" instead of "enabled" in the spec, but that's the shared naming they decided :/
Disabling use case when it's reporting wrong IPs for example ?

Reply:
Why would wrong IPs be reported, and why would we not send them anyway?

Reply:
The algorithm is quite heuristic, so we can't be sure we're not doing something wrong, hence letting the user choose to disable it. What if the user has something sensitive inside one of these picked headers uh